src/components/charts/timebank-piechart.tsx

```diff
@@ -42,15 +42,14 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
           outerRadius={50}
           label={(entry) => {
             if (!entry.dataKey) {
-              return null; 
-            } else {
-              return renderCustomizedLabel(entry);
+              return null;
             }
+            return renderCustomizedLabel(entry);
           }}
           labelLine={false}
-        >     
+        >
           {dailyEntryToChart(personDailyEntry).map((_entry, index) => (
-            <Cell key={`cell-${index}`} fill={COLORS[index]} />
+            <Cell key={`cell-${_entry.name}`} fill={COLORS[index]} />
```

Comment:
Are you sure entry.name is unique. Do we have entry.id?
Also _ is used to indicate an unused argument, as it is being used here now we can remove the _

Reply:
There is no entry.id property